src/Libraries/CoreNodeModels/DynamoConvert.cs

```diff
@@ -86,6 +91,23 @@ public ConversionUnit SelectedToConversion
             }
         }
 
+        /// <summary>
+        /// Conversion factor of from unit and to to unit.
+        /// </summary>
+        [JsonProperty("ConversionFactor")]
```

Comment:
Wait. You are right. `MetricConversion` is a UI property (Length, Area, Volume). It should be ignored.